tools/ncbi_blast_plus/README.rst

```diff
@@ -136,6 +136,7 @@ a galaxy specific suffix which gets reset to zero with each new BLAST version:
 ============== ===============================================================
 Version        Changes
 -------------- ---------------------------------------------------------------
+2.10.1+galaxy1 - Make cached reference data selector non-optional
```

Comment:
cached reference data -> locally installed database?

And a full stop for consistency ;)